x-pack/legacy/plugins/lens/public/app_plugin/app.test.tsx

```diff
@@ -7,21 +7,18 @@
 import React from 'react';
 import { App } from './app';
 import { EditorFrameInstance } from '../types';
-import { Chrome } from 'ui/chrome';
-import { toastNotifications } from 'ui/notify';
 import { Storage } from 'ui/storage';
 import { Document, SavedObjectStore } from '../persistence';
 import { mount } from 'enzyme';
 import { QueryBarTopRow } from '../../../../../../src/legacy/core_plugins/data/public/query/query_bar';
 import { SavedObjectsClientContract } from 'src/core/public';
+import { coreMock } from 'src/core/public/mocks';
```

Comment:
Nice, that makes it much easier to test code integrating with the rest of Kibana.